code/datums/weather/weather_types/snow_storm.dm

```diff
@@ -27,3 +27,25 @@
 /datum/weather/snow_storm/weather_act(mob/living/L)
 	L.adjust_bodytemperature(-rand(5,15))
 
+
+// since snowstorm is on a station z level, add extra checks to not annoy everyone
+/datum/weather/snow_storm/can_get_alert(mob/player)
+	var/standard_check = ..()
```

Comment:
You should just `if (!..())` instead.

Reply:
then how will it return the value of ..()

Reply:
and since the value of ..() is only used for the if check and the first return it seemed logical to name the var in a way to indicate what it represents

Reply:
```dm
if (!..())
    return FALSE
```